releases/23.0.0.3/full/Dockerfile.ubi.ibmjava8

```diff
@@ -1,4 +1,4 @@
-FROM ibmjava:8-ubi
+FROM icr.io/appcafe/ibmjava:8-sdk-ubi8
```

Comment:
The existing base image`ibmjava:8-ubi` is built locally and it includes a JRE, so we should continue to use JRE

It's done here: https://github.ibm.com/was-docker/build-liberty-images-ubi/blob/b1df80fdc0173f3dbed8994d3e9e7d4cb9051ccb/build.sh#L62

using this Dockerfile: https://raw.githubusercontent.com/ibmruntimes/ci.docker/main/ibmjava/8/jre/ubi/Dockerfile

Also, `icr.io/appcafe/ibmjava:8-sdk-ubi8` only supports amd64. Same is true for `icr.io/appcafe/ibmjava:8-jre-ubi8`